sdk/storage/Azure.Storage/Azure.Storage.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>12.1.0-preview.1</Version>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <NoWarn>$(NoWarn);NU5128</NoWarn>
+
+    <PackageType>meta-package</PackageType>
+    <PackageTags>Storage Blobs</PackageTags>
```

Comment:
Are these tags intentional? Should you add other common tags like `windowsazureofficial`?

Reply:
Yes I will add the common tags to this now as well, https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Directory.Build.Data.props#L119. When I did this work initially those weren't ready yet.